src/rt/backtrace/dwarf.d

```diff
@@ -533,6 +605,148 @@ long readSLEB128(ref const(ubyte)[] buffer) @nogc nothrow
     return val;
 }
 
+Array!EntryFormatData readEntryFormat(ref const(ubyte)[] buffer, ref Array!ulong entryFormat) @nogc nothrow
+{
+    // The count needs to be pair, as the specification says
+    assert(entryFormat.length % 2 == 0);
+    Array!EntryFormatData result;
+
+    for (uint i = 0; i < entryFormat.length; i += 2)
+    {
+        EntryFormatData efdata;
+        ulong form = entryFormat[i + 1];
+
+        switch (entryFormat[i])
+        {
+            case StandardContentDescription.path:
+                if (form == FormEncoding._string)
+                    efdata.path = buffer.readString();
+                else
+                {
+                    size_t offset = buffer.read!size_t;
+
+                    // TODO: set filename.path to the string at offset
```

Comment:
Is this necessary for proper functionality?

Reply:
I'll need this later to imprive the runtime debug informations. As this part is not used for now, shall I remove it ?

Reply:
No its fine to leave it there. I was wondering whether or not it was require for this diff to function correctly.